src/yt_dlp.gd

```diff
@@ -173,6 +180,25 @@ class Download extends RefCounted:
 		var output: Array = []
 		OS.execute(executable, PackedStringArray(options_and_arguments), output)
 		
+		if _convert_to_video && _video_format == Video.OGV:
```

Comment:
Please avoid C style syntax, prefer `and`.

Reply:
Sure thing